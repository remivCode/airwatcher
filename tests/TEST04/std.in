username;
password;
1;
5;
10;
1;
2024;
5;
4;
12;
5;
10;
1;
2024;
5;
5;
12;
5;
10;
1;
2024;
5;
5;
12;
