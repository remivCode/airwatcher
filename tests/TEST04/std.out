TEST EXECUTION
----- AUTHENTIFICATION PHASE -----

0. End
1. Authentification

If you already have an account, please enter your username and your password:
Username: 
Password: 

Enter the user (1: Government, 2: Private, 3: Provider)
Enter a number: 
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice: 

Enter the GPS coordinates
Enter latitude: 
Enter longitude: 
Enter the date
Enter year: 
Enter month:
Enter day:
Enter hour:
----- Air Quality Measurement -----
ATMO Index: 1
O3: 13
SO2: 18
NO2: 12
PM10: 1
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice: 

Enter the GPS coordinates
Enter latitude: 
Enter longitude: 
Enter the date
Enter year: 
Enter month:
Enter day:
Enter hour:
----- Air Quality Measurement -----
ATMO Index: 2
O3: 32
SO2: 30
NO2: 24.5
PM10: 13.5
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice:

Enter the GPS coordinates
Enter latitude: 
Enter longitude: 
Enter the date
Enter year: 
Enter month:
Enter day:
Enter hour:
----- Air Quality Measurement -----
ATMO Index: 1
O3: 10
SO2: 15.5
NO2: 8.5
PM10: 1.5
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice: 

----- AUTHENTIFICATION PHASE -----

0. End
1. Authentification
